### PR TITLE
[DOC][Kommander] RN's for Kommander 1.3.3 patch.

### PR DESCRIPTION
**Pull request by russurquhart1**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Description of changes being made
RN docs for 1.3.3 Patch

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Bug fixes
+
+- Updated UI to handle new KUDO param types.
+- Duplicate namespaces are no longer created for a single workspace.
```
Comment (cneth) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:25]: This item was also already published in the Kommander 1.3.2 RN, so shouldn't be in these.

Are there any actual fixes in Kommander 1.3?  Or is it just a component bumps?

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Bug fixes
+
+- Updated UI to handle new KUDO param types.
```
Comment (cneth) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:24]: This release note is also in the 1.3.2 RN, and seems to refer to https://jira.d2iq.com/browse/D2IQ-74749, which was already released in 1.3.2.  So I am not sure why we are including it here.

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md (hunk 2 above)
Comment (cmastr) [reply pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:24]: remove this line.

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Bug fixes
```
Comment (cmastr) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:22]: Do any of these bug fixes have COPS ticket numbers @russurquhart1

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+**D2iQ&reg; Kommander&reg; version 1.3.3 was released on 9, June 2021.**
+
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
```
Comment (makkes) [inline pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:20]: It feels a little weird having this same sentence above all release notes for 1.3 patch release. As a user I would've expected a summary of the actual changes that ship with this release and not a generic product description.

File: pages/dkp/kommander/1.3/release-notes/1.3.3/index.md (hunk 1 above)
Comment (russurquhart1) [reply pages/dkp/kommander/1.3/release-notes/1.3.3/index.md:25]: This was kind of a template RN, with those items to be deleted or updated! I'm removing this PR. Grace and I have another PR that hits all the points of the RN. Thx

**Comment by russurquhart1** [comment]
Refer to https://github.com/mesosphere/dcos-docs-site/pull/3635

**Pull request closed without merging**